_pages/en_US/installing-boot9strap-(mset9-cli).txt

```diff
@@ -112,7 +130,12 @@ Do NOT skip this section! If you skip it, applications may crash unexpectedly an
     + **macOS**: Double-click `mset9.command` and enter your password if prompted
     + **Linux**: open a Terminal window, `cd` to the root of your SD card, then type `python3 mset9.py` and press Enter
 1. Type the number corresponding to your console model and version, then press Enter
-1. In the MSET9 window, type `3`, then press Enter to remove MSET9
+    + The current state should display [Injected](/images/screenshots/mset9/mset9-injected.png)
+    + If you have already removed the trigger file (or never injected in the first place), the current state will show [Ready](/images/screenshots/mset9/mset9-ready.png), skip to Step 6
+1. Type `4`, then press Enter to remove the trigger file
+    + You should see "Removed trigger file."
+    + If there is not an option `4` then proceed to Step 6
```

Comment:
this sequence reads as overly confusing to me what with the multiple "skip to Step 6"s but again, not entirely sure of the best way to resolve this

Reply:
Yeah this was trying to cover every possible situation.

We could just write section 4 as if you're 100% following the guide and see if edge cases pop up in discord later.